Source/UI/NewUI/PaymentForm/View/PaymentHeaderView.swift

```diff
@@ -86,11 +90,25 @@ public final class PaymentHeaderView: UIView {
 extension PaymentHeaderView {
 
   private func setupViewsInOrder() {
+    setupBackgroundView()
     setupMainStackView()
     setupIconStackView()
     addArrangedSubview()
   }
 
+  private func setupBackgroundView() {
+    addSubview(backgroundView)
+    NSLayoutConstraint.activate([
+      backgroundView.topAnchor.constraint(equalTo: topAnchor,
+                                          constant: -Constants.Padding.l.rawValue),
```

Comment:
What is `l` ? in Constants.Padding.l. Can you rename it to having a longer name for readability ?

Reply:
This has already been used extensively in the project so could be a larger refactor if done. Wouldn't want that to happen here as we cannot measure.

The Padding enum uses abbreviated common form variables to control number of paddings:

```swift
    public enum Padding: Double {

        /// parameter size: 2
        case xxs = 2

        /// parameter size: 8
        case xs = 8

        /// parameter size: 10
        case s = 10

        /// parameter size: 16
        case m = 16

        /// parameter size: 20
        case l = 20

        /// parameter size: 24
        case xl = 24

        /// parameter size: 28
        case xxl = 28

        /// parameter size: 40
        case xxxl = 40
    }
```

Might need to discuss and plan a route if we think an improvement is necessary, but for this PR this should not be relevant :)

Reply:
Thanks, Something to add in backlog for future improvement if needed.